plasmapy/formulary/dielectric.py

```diff
@@ -3,17 +3,22 @@
     "cold_plasma_permittivity_SDP",
     "cold_plasma_permittivity_LRP",
     "permittivity_1D_Maxwellian",
+    "RotatingTensorElements",
+    "StixTensorElements",
```

Comment:
I needed to expose these to the public API so that they could be used as return annotations without causing a "missing reference" error when doing the doc build.

Reply:
Could you explain this when we next talk please?

Reply:
Would be happy to, with the caveat that I only understand 83% of what's happening here!